### ISSUE TITLE
request to add code for Chorei? 

### ISSUE THREAD

**Issue by quail4sale** [glottolog/glottolog#774]
Chorei is a Tibeto-Burman language spoken in the Karimganj district of Assam (and somewhat in Mizoram, Manipur, and Tripura) with approximately 5000–6000 speakers. Here is some literature that references Chorei: 

- Haokip, Pauthang. 2018. Agreement in Kuki-Chin languages of Barak valley. _Journal of South Asian Languages and Linguistics, 5_(2), 159-210. doi: 10.1515/jsall-2018-0008.
- Haokip, Pauthang. 2019. Kuki-Chin phonology of Barak Valley. _Indian Linguistics, 80_(3-4), 85-114. issn: 0378-0759. 
- Moseley, Christopher. 2007. _Encyclopedia of the World's Endangered Languages_ (1st ed.). Routledge. doi: 10.4324/9780203645659. (briefly mentioned on page 327 in van Driem's chapter). 

The first two sources provide some details of Chorei's phonology and agreement system. In Haokip (2019), Chorei is mislabeled as ISO 639-3 cfm (Falam Chin), but Haokip has since noted that this is misleading, as Chorei is not mutually intelligible with Falam Chin. He writes that Chorei is closer to Hrangkhol, but still Chorei has a different phonemic inventory. 

I will soon be adding some Chorei material to this collection: https://digital.library.unt.edu/explore/collections/LGBV/ 
and need a unique code to identify Chorei. Is it possible to add a code for Chorei? 

-Mary

**Comment by d97hah** [comment]
Hi! Haokip (2018, 2019) says Chorei is intelligible with Sakachep (not
Hrangkhol) and Sakachep and Chorei do have identical phonemic inventories
in Haokip 2019. Consequently Chorei references such as Haokip (2018) are
filed with Sakachep, but if you need a dialect-level glottocode for Chorei
I've made one in a new PR. all the best, H

Pada tanggal Sel, 16 Nov 2021 pukul 21.17 quail4sale <
***@***.***> menulis:

**Comment by quail4sale** [comment]
Apologies for my confusion! I couldn't speak to the status of Chorei exactly--I am requesting the code on Haokip's behalf--but yes, a dialect code works perfectly for archiving purposes. I appreciate it :) 
-Mary

**Comment by xrotwang** [comment]
So this will be chor1278 to be released with Glottolog 4.5 in a week or so.

```
$ glottolog tree saka1283
INFO    glottolog/glottolog at /home/robert_forkel/projects/glottolog/glottolog
Sino-Tibetan [sino1245]
   └─ Kuki-Chin-Naga [kuki1245]
      └─ Kuki-Chin [kuki1246]
         └─ Central Kuki-Chin [cent2330]
            └─ Core Central Kuki-Chin [cent2005]
               └─ Mizoic [mizo1244]
                  └─ Hmaric [hmar1240]
                     └─ Sakachep-Chorei [saka1283]
                        ├─ Chorei [chor1278]
                        └─ Sakachep [sake1246]
```
